routers/likes.py

```
@@ -0,0 +1,4 @@
+from fastapi import APIRouter
+
+asasd
```

Ruff (0.14.11)

Found useless expression. Either assign it to a variable or remove it.

(B018)